### ISSUE TITLE
Include Web Support For Compose Module

### ISSUE THREAD

**Issue by wakaztahir** [icerockdev/moko-resources#422]
here's a sample that I use , that works on my web app here https://qawaz.github.io/sketchable-web , this will only load the file once , more requests will return with empty strings until the resource has loaded

```kotlin

private var loader: JsStringProvider? by mutableStateOf(null)
private var isLoading = false

private val WebExceptionHandler: (Throwable) -> Unit = {
    it.printStackTrace()
}

@Composable
actual fun stringResource(res: StringResource): String {
    return if (loader == null && !isLoading) {
        isLoading = true
        LaunchedEffect(null) {
            try {
                loader = res.loader.getOrLoad()
            } catch (e: Throwable) {
                WebExceptionHandler(e)
            }
            isLoading = false
        }
        ""
    } else {
        return loader?.let { load ->
            try {
                res.localized(load, null)
            } catch (e: Throwable) {
                WebExceptionHandler(e)
                null
            }
        } ?: "fail"
    }
}

@Composable
actual fun loadAsText(asset: AssetResource): String {
    return produceState("") {
        value = asset.getText()
    }.value
}
```

**Comment by wakaztahir** [comment]
loadAsText is similar for other modules i.e `jvmMain` and `androidMain` , so I think it should be added there as well

I could create a pull request but this project configuration is bloated , resources file with a path to mpp-library's resources , it doesn't work on windows , kotlin version 1.6.10 causes error on windows with Projects must be configuring , plus compose-jb version is outdated , Kotlin version should be 1.8.0 and compose version should be 1.3.0

**Comment by Alex009** [comment]
will be released in 0.21.0
